docs/sdk/v3/guides/liquidity/03-checking-collecting-fees.md

```diff
@@ -1,6 +1,57 @@
 ---
 id: liquidity-fees
-title: Checking & Collecting Fees
+title: Collecting Fees
 ---
 
-TODO
\ No newline at end of file
+## Introduction
+
+This guide will cover how to collect fees from a liquidity position on the Uniswap V3 protocol.
+It is based on the [collecting fees code example](https://github.com/Uniswap/examples/tree/main/v3-sdk/collecting-fees), found in the Uniswap code examples [repository](https://github.com/Uniswap/examples).
+To run this example, check out the examples's [README](https://github.com/Uniswap/examples/blob/main/v3-sdk/collecting-fees/README.md) and follow the setup instructions.
+
+:::info
+
+If you need a briefer on the SDK and to learn more about how these guides connect to the examples repository, please visit our [background](./01-background.md) page!
+
+:::
+
+In the Uniswap V3 protocol, liquidity positions are represented using non-fungible tokens. In this guide we will use the `NonfungiblePositionManager` class to help us mint a liquidity position for the  **USDC - DAI** pair. We will then attempt to collect any fees that the position has accrued from those trading against our provisioned liquidity. The inputs to our guide are the **two tokens** that we are pooling for, the **amount** of each token we are pooling for, the Pool **fee** and the **max amount of accrued fees** we want to collect for each token.
+
+The guide will **cover**:
+1. Collecting accrued fees from our position
+
+Note that the minting logic is not covered in this guide as it was covered in detail in the [previous guide](./01-minting-position.md).
+
+At the end of the guide, given the inputs above, we should be able to mint a liquidity position with the press of a button and view the position's id on the UI of the web application. We should also be able to collect the accrued fees (if any) with the press of a button and see the change reflected in the balance of our tokens.
+
+## Example
+
+### Collecting accrued fees from our position
+
+All of the fee collecting logic can be found in the [`collectFees`](https://github.com/Uniswap/examples/blob/be67e7df220b0a270c9d18bbaab529e017213adf/v3-sdk/collecting-fees/src/example/Example.tsx#L24) function. The function is very similar to `mintPosition`, except for some details that we will go over. Notice how the **Collect Fees** button is disabled until a position is minted.
+
+Note that we do not need to give approval to the `NonfungiblePositionManager` to transfer our tokens as we have already done that when minting our position. 
```

Comment:
This feels like it would fit better at the end - I don't yet know why we would need to do this if I haven't been shown any code